spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -212,43 +236,57 @@ public class MyClass {
 
 This class would result in the following metrics being added.
 
-* `ft.com.example.MyClass.doWork.invocations.total`
-* `ft.com.example.MyClass.doWork.invocations.failed`
-* `ft.com.example.MyClass.doWork.retry.callsSucceededNotRetried.total`
-* `ft.com.example.MyClass.doWork.retry.callsSucceededRetried.total`
-* `ft.com.example.MyClass.doWork.retry.callsFailed.total`
-* `ft.com.example.MyClass.doWork.retry.retries.total`
-* `ft.com.example.MyClass.doWork.timeout.executionDuration`
-* `ft.com.example.MyClass.doWork.timeout.callsTimedOut.total`
-* `ft.com.example.MyClass.doWork.timeout.callsNotTimedOut.total`
+```
+ft.invocations.total{method="com.example.MyClass.doWork", result="valueReturned", fallbackUsed="none"}
+ft.invocations.total{method="com.example.MyClass.doWork", result="exceptionThrown", fallbackUsed="none"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="true", retryResult="valueReturned"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="true", retryResult="exceptionNotRetryable"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="true", retryResult="maxRetriesReached"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="true", retryResult="maxDurationReached"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="false", retryResult="valueReturned"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="false", retryResult="exceptionNotRetryable"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="false", retryResult="maxRetriesReached"}
+ft.retry.calls.total{method="com.example.MyClass.doWork", retried="false", retryResult="maxDurationReached"}
+ft.retry.retries.total{method="com.example.MyClass.doWork"}
+ft.timeout.calls.total{method="com.example.MyClass.doWork", timedOut="true"}
+ft.timeout.calls.total{method="com.example.MyClass.doWork", timedOut="false"}
+ft.timeout.executionDuration{method="com.example.MyClass.doWork"}
+```
 
 Now imagine the `doWork()` method is called and the invocation goes like this:
 
 * On the first attempt, the invocation takes more than 1000ms and times out
 * On the second attempt, something goes wrong and the method throws an `IOException`
 * On the third attempt, the method returns successfully and the result of this attempt is returned to the user
```

Comment:
Is this attempt from the Retry operation?

Reply:
Yes, I'll see if I can reword this to make it clearer that it's doing retries.

Reply:
I've updated this example to make it explicit that it's doing retries.